run-videochat.sh

```diff
@@ -41,7 +41,9 @@
 # - souphttpsrc version 1.0.6
 # - v4l2sink version 1.0.6
 # - v4l2loopback version 0.7.0
-
+#to fix no pulseaudio daemon running in arch/manjaro
+killall pulseaudio
```

Comment:
This may be problematic in other systems, as it may stop running audio playback at the time. I'd prefer this to be conditional to some check, rather than always restarting it.